docs/keyring-core.md

```diff
@@ -88,7 +88,7 @@ export interface KeyringContextActions {
    * storage. Use cancelRegisterSpace to abort. Automatically sets the
    * newly registered space as the current space.
    */
-  registerSpace: (email: string) => Promise<void>
```

Comment:
we may still want to allow passing a parameter here to indicate which account you want to use to add the storage provider to the space

Reply:
(e.g. you could have claimed delegations authorizing your issuer `did:key` to act as more than one account)

Reply:
Yep - it's not documented here but `registerSpace` also takes a map of options and I think that would be a better place for an email parameter if we do end up adding it to the lower level API - since the default case for this API is to just use the account we most recently `authorized` against I'd like to optimize for that both in the implementation and documentation for now.

Do you have any thoughts on user stories that might use the functionality you're proposing? I think in most cases where we'd be acting for more than one account we'd fire up two `Agent`s and authorize with them both separately (I'm imagining an "account selector" kind of interface).